solutions/search/esql-for-search.md

```diff
@@ -16,162 +16,112 @@ products:
 
-### Relevance scoring [esql-for-search-scoring]
+**Searching** both filters documents and ranks them by relevance. It calculates a score for each matching document based on how well the content matches your query, allowing you to sort results from most relevant to least relevant. Search functions use advanced text analysis including stemming, synonyms, and fuzzy matching.
 
-To get the most relevant results first, you need to use `METADATA _score` and sort by score. For example:
+**When to choose filtering:**
+- Exact category matches (`category.keyword == "Electronics"`)
+- Date ranges (`date >= "2023-01-01"`)
+- Numerical comparisons (`price < 100`)
+- Any scenario where you want all matching results without ranking
 
-```esql
-FROM books METADATA _score
-| WHERE match(title, "Shakespeare") OR match(plot, "Shakespeare")
-| SORT _score DESC
-```
+**When to choose searching:**
+- Text queries where some results are more relevant than others
+- Finding documents similar to a search phrase
+- Any scenario where you want the "best" matches first
 
-### How `_score` works [esql-for-search-how-scoring-works]
+{{esql}}'s search functions address several key limitations that existed for text filtering: they work directly on multivalued fields, leverage analyzers for proper text analysis, and use optimized Lucene index structures for better performance.
 
-When working with relevance scoring in {{esql}}:
+### Relevance scoring
 
-* If you don't include `METADATA _score` in your query, this only performs filtering operations with no relevance calculation.
-* When you include `METADATA _score`, any search function included in `WHERE` conditions contribute to the relevance score. This means that every occurrence of `MATCH`, `QSTR` and `KQL` will affect the score.
-* Filtering operations that are not search functions, like range conditions and exact matches, don't affect the score.
-* Including `METADATA _score` doesn't automatically sort your results by relevance. You must explicitly use `SORT _score DESC` or `SORT _score ASC` to order your results by relevance.
+To get relevance-ranked results, you must explicitly request scoring with `METADATA _score` and sort by the score. Without this, even search functions like `MATCH` will only filter documents without ranking them.
 
-## Full text search [esql-for-search-full-text]
+**Without `METADATA _score`**: All operations are filtering-only, even `MATCH`, `QSTR`, and `KQL` functions. Documents either match or don't match - no ranking occurs.
 
-### `MATCH` function and operator [esql-for-search-match-function-operator]
+**With `METADATA _score`**: Search functions contribute to relevance scores, while filtering operations (range conditions, exact matches) don't affect scoring. You must explicitly use `SORT _score DESC` to see the most relevant results first.
```

Comment:
I think adding a link to the search functions would be great, so users can check which functions contribute to score.

```suggestion
**With `METADATA _score`**: Search functions (`MATCH`, `QSTR`, `KQL`..) contribute to relevance scores, while filtering operations (range conditions, exact matches) don't affect scoring. You must explicitly use `SORT _score DESC` to see the most relevant results first.
```